packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistLibrary.tsx

```diff
@@ -55,31 +55,40 @@ export const PlaylistLibrary = (props: PlaylistLibraryProps) => {
   )
 
   return (
-    <Droppable
-      className={styles.droppable}
-      hoverClassName={styles.droppableHover}
-      onDrop={handleDrop}
-      acceptedKinds={acceptedKinds}
-    >
-      <GroupHeader
-        className={cn(styles.header, {
-          [styles.droppableLink]: draggingKind === 'playlist'
-        })}
-      >
-        {messages.header}
-        <CreatePlaylistLibraryItemButton scrollbarRef={scrollbarRef} />
-      </GroupHeader>
-      {!library || isEmpty(library?.contents) ? (
-        <EmptyLibraryNavLink />
-      ) : (
-        library.contents.map((content) => (
-          <PlaylistLibraryNavItem
-            key={keyExtractor(content)}
-            item={content}
-            level={0}
-          />
-        ))
+    <ClassNames>
+      {({ css }) => (
+        <Droppable
+          className={css({
+            borderRadius: 6,
+            paddingTop: -10,
+            marginTop: -10,
+            transition: `background ${motion.quick}`
+          })}
+          hoverClassName={css({ backgroundColor: 'rgba(152, 73, 214, 0.15)' })}
```

Comment:
maybe check with design if there is a standard color we want to do here. one negative with this value is that it doesn't respect light vs dark vs matrix

Reply:
Am going to redo this with a pseudo element that I can apply opacity to. It's pretty close to one of our secondary shades, just with opacity applied.